ccp/src/proof_storage.rs

```diff
@@ -81,14 +81,9 @@ impl ProofStorageDrainer {
 
     pub async fn validate_proofs(
         &mut self,
-        epoch_params: Option<&EpochParameters>,
+        epoch: &Option<EpochParameters>,
```

Comment:
This looks unrelated.

Reply:
otherwise formatting will be not the best